test/node-binding.test.cjs

```diff
@@ -133,3 +134,39 @@ test("Returns meaningful error if file does not exist and noFailOnFsErrors", asy
   t.is(reason, "file-not-exists");
   t.is(file, path.join(IMAGES_PATH, "not-existing.png"));
 });
+
+test("Accepts buffer as input for compare image", async (t) => {
+  const buffer = await readFile(path.join(IMAGES_PATH, "donkey-2.png"), { encoding: "binary" });
+  const { reason, diffCount, diffPercentage } = await compare(
+    path.join(IMAGES_PATH, "donkey.png"),
+    buffer,
+    path.join(IMAGES_PATH, "diff.png"),
+    {
+      __binaryPath: BINARY_PATH,
+      compareImageType: "png"
+    }
+  );
+
+  t.is(reason, "pixel-diff");
+  t.is(diffCount, 109861);
+  t.is(diffPercentage, 2.85952484323);
+});
+
+test.skip("Accepts buffer as input for base and compare image at the same time", async (t) => {
```

Comment:
This test is currently failing, as we can't distinguish from nodes stdin stream, where the first image ends and the second one starts.